build.js

```diff
@@ -1,11 +1,12 @@
 const path = require("path")
 const fs = require("fs")
 const process = require("process")
-const size = require('human-format');
+const size = require("human-format")
```

Comment:
Is this `require` needed? It seems like the one three lines below is what is being used.

Reply:
You're right, I brought it in twice. Thanks!